### ISSUE TITLE
footnotes in table are ignored

### ISSUE THREAD

**Issue by kmuto** [Trimmarks/versatype-issues#62]
(I'm not sure whether this is a issue caused by viliostyle.js.)

In v2017.06, the footnotes in the table  are ignored.

Here is a sample.

```
<html>
<head>
<style>
body {
  counter-reset: footnote 0;
}

td {
  border: solid 1px;
}

span.footnote {
  float: footnote;
  color: red;
  counter-increment: footnote;
  font-size: 0.7em;
  text-align: left;
}

.footnote::footnote-call {
  content: "*" counter(footnote,decimal);
  font-size: 0.7em;
  vertical-align: super;
  position: relative;
  top: -0.1em;
  white-space: nowrap;
}

.footnote::footnote-marker {
  content: "*" counter(footnote,decimal);
  font-size: 0.7em;
}
</style>
</head>
<body>
<p>STR<span class="footnote" id="fn1">FOOTNOTE1</span></p>
<p>STR2<span class="footnote" id="fn2">FOOTNOTE2</span></p>
<table>
<tr><td>A<span class="footnote" id="fn3">FOOTNOTE3</span></td><td>B</td></tr>
<tr><td>C</td><td>D<span class="footnote" id="fn4">FOOTNOTE4</span></td></tr>
</table>
<p>STR3<span class="footnote" id="fn5">FOOTNOTE5</span></p>
</body>
</html>
```
[vivliostyle-output-201610.pdf](https://github.com/vivliostyle/vivliostyle-formatter-issues/files/1096926/vivliostyle-output-201610.pdf)
[vivliostyle-output-201706.pdf](https://github.com/vivliostyle/vivliostyle-formatter-issues/files/1096927/vivliostyle-output-201706.pdf)

**Comment by MurakamiShinyu** [comment]
This issue was moved to vivliostyle/vivliostyle.js#438

**Comment by kwkbtr** [comment]
This bug has been fixed in VersaType Converter 2018.5.
